user-guide/Troubleshooting/Procedures/Cloud_Connection_Issues.md

```diff
@@ -0,0 +1,75 @@
+---
+uid: Cloud_Connection_Issues
+---
+
+# Investigating dataminer.services connection issues
+
+This page provides solutions for potential connectivity problems with [dataminer.services](xref:AboutCloudPlatform).
+
+## Remote access issues
+
+If you encounter issues with dataminer.services features that may be caused by connection issues, follow the actions below in the specified order:
+
+1. [Check the status of the DxMs](#check-the-status-of-the-dxms).
+1. [Check the status of the DMS](#check-the-status-of-the-dms).
+1. [Check the cloud session](#check-the-cloud-session)
+1. [Make sure requirements are met](#make-sure-requirements-are-met)
+
+### Check the status of the DxMs
+
+A common cause for issues with dataminer.services features is a problem with the connection from the [DxMs](xref:DataMinerExtensionModules) towards dataminer.services.
+
+You can resolve this as follows:
+
+1. Verify whether all installed [DxMs](xref:DataMinerExtensionModules) are running, otherwise start them.
+
+1. Make sure the most recent versions of the DxMs are installed. See [Upgrading nodes to the latest DxM versions](xref:Managing_cloud-connected_nodes#upgrading-nodes-to-the-latest-dxm-versions).
+
+   This way, you avoid dealing with issues that have already been fixed.
+
+1. Verify the accessibility of the DMA host:
```

Comment:
This doesn't really seem to be related to the status of the DxMs. I think we should move this step and its substeps to the "Check the status of the DMS" section below, as the description of how the user can check whether the DMA is running. The existing info there can then be added at the end and rephrased as `If the page does not load correctly, this indicates a problem with the DMA host. Restart the DMA to resolve this. You can do so using the [DataMiner Taskbar Utility](xref:DataMiner_Taskbar_Utility).`